README.md

```diff
@@ -6,8 +6,22 @@ Space time benchmarking project with k6 tool
  - The entry points for the tests need to have "test" word in the name to distinguish them from auxiliary files.
 
 ## Running tests
+
+### Prerequisites
+
  - Make sure Kibana is running and eCommerce sample data is loaded
- - Install dependencies and transpile typescript code into JS and bundle the project: `yarn install && yarn webpack`
+
+### Directly with k6
+
+ - Install dependencies and transpile typescript code into JS and bundle the project: `make build`
  - Run your test:
     - using kibana.json in assets folder: `k6 run -e USE_KIBANA_JSON=1 dist/login_test.js`
     - by passing values directly: `k6 run -e KIBANA_USERNAME=elastic -e KIBANA_PASSWORD=changeme -e KIBANA_URL="http://localhost:5620" -e KIBANA_VERSION=8.3.0 dist/login_test.js`
+## Running tests directly with k6
+
+ - Ensure that Vault is properly setup
```

Comment:
I would add a line about GO installation and `export PATH=$PATH:$(go env GOPATH)/bin`

Reply:
Added in https://github.com/elastic/kibana-k6-benchmarking/pull/12/commits/e42e62c55b8034e44e2695a78e0c8dc551819d09.